wiki/en/misc/1-index.html

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jamulus ‒ Play music online. With friends. For free."
+title: "Jamulus — Play music online with friends."
 lang: "en"
 permalink: /
 layout: mainhomepage
 ldjAppCategory: "Music"
-ldjSlogan: "Play music online. With friends. For free."
+ldjSlogan: "Play music online with friends."
 ldjCreator: "Volker Fischer and contributors"
-metadescription: "Jamulus is open source software which enables musicians to perform music, rehearse or just jam, all in real time over the Internet."
+metadescription: "Copylefted libre software for musicians to perform, rehearse or just jam — all in real time over the Internet."
 mAltProgIcon: "Jamulus icon"
-mTSlogan: "Play music online. With friends. For free."
+mTSlogan: "Play music online. With friends."
```

Comment:
```suggestion
mTSlogan: "Play music online with friends."
```

Reply:
I don't like "hip speak" like these brief sentences, but I like featuring the price. That three-sentence slogan appears in the browser's page title, then again on the page prominently, and in the search result titling as well. It is indeed the product's top marketing slogan.

Reply:
>  It is indeed the product's top marketing slogan.

Yes. No cost is also a "selling point". The sentence is NOT about the license or philosophy and thus unrelated.

Reply:
What sounds more like marketing, and is Jamulus good at that sort of thing?